app/server/lib/NSandbox.ts

```diff
@@ -41,28 +41,44 @@ type SandboxMethod = (...args: any[]) => any;
  * in tests) which runs grist/main.py.
  */
 export interface ISandboxOptions {
-  command?: string;       // External program or container to call to run the sandbox.
-  args: string[];         // The arguments to pass to the python process.
-
-  preferredPythonVersion?: string;  // Mandatory for gvisor; ignored by other methods.
+  // External program or container to call to run the sandbox.
+  command?: string;
+  // The arguments to pass first to the sandbox process.
+  sandboxArgs: string[];
```

Comment:
Per our conversation, sandboxArgs and pythonArgs are present basically for testing at this point, could you call them testSandboxArgs and testPythonArgs and make them optional?

Reply:
Renamed!